core/src/main/java/org/apache/accumulo/core/spi/balancer/GroupBalancer.java

```diff
@@ -213,7 +213,7 @@ public long balance(BalanceParameters params) {
       return 5000;
     }
 
-    final DataLevel currentLevel = DataLevel.valueOf(params.currentLevel());
+    final DataLevel currentLevel = DataLevel.valueOf(params.partitionName());
```

Comment:
since `valueOf` is an enum method, won't this throw an `IllegalArgumentException` if params has a custom partitionName?

Reply:
I think this comment is against an older commit, but not sure.  When I look at that line in the latest code do not see that.